apps/pr-deploy-site/config/pre-copy.json

```diff
@@ -1,7 +1,17 @@
 {
   "copyTo": {
-    "dist": ["./node_modules/@uifabric/*/dist/**", "index.html"],
-    "dist/todo-app": ["./node_modules/todo-app/dist/**"],
-    "dist/test-bundles": ["./node_modules/test-bundles/dist/**"]
+    "dist": [
```

Comment:
Kind of a nitpicky thing but can you install the Prettier extension in your VS Code and re-save these files? I think all the formatting changes are because you don't have the extension installed, and it adds a lot of extra noise to PRs and history. (I recommend disabling it globally and enabling it per-workspace to prevent unwanted changes in other projects--its default setting of forcing 80-char lines is pretty obnoxious.)

Reply:
As discussed offline, prettier does and should run as part of pre-commit task.